### PR TITLE
Add latest trades and quotes

### PR DESCRIPTION
**Pull request by kalmant**

### CODE REVIEW

File: alpaca/alpaca_test.go [hunk 1]
```diff
@@ -265,6 +266,106 @@ func (s *AlpacaTestSuite) TestAlpaca() {
...
+		}
+		do = func(c *Client, req *http.Request) (*http.Response, error) {
+			return &http.Response{
+				Body: ioutil.NopCloser(strings.NewReader(latestTradeJSON)),
+			}, nil
+		}
+
+		actualLatestTrade, err := GetLatestTrade("AAPL")
+		assert.NotNil(s.T(), actualLatestTrade)
+		assert.Nil(s.T(), err)
```
Comment (gnvk) [inline alpaca/alpaca_test.go:305]: [minor] `NoError` is preferable

Comment (gnvk) [reply alpaca/alpaca_test.go:305]: same for a few more places in this file

File: alpaca/alpaca_test.go [hunk 2]
```diff
@@ -265,6 +266,106 @@ func (s *AlpacaTestSuite) TestAlpaca() {
 		assert.Nil(s.T(), actualLastTrade)
 	}
 
+	// get latest trade
```
Comment (gnvk) [inline alpaca/alpaca_test.go:269]: I'd prefer to move this to a separate test case but agree with you to stay consistent with the rest of the file

**Review by gnvk — APPROVED** [review]
LGTM

File: alpaca/alpaca_test.go (hunk 1 above)
Comment (kalmant) [reply alpaca/alpaca_test.go]: You are right, I fixed it throughout the file.

File: alpaca/alpaca_test.go (hunk 2 above)
Comment (kalmant) [reply alpaca/alpaca_test.go:269]: Yeah, I did not want to change things in this PR

**Pull request merged**